From cb9243176e69b534db571369fa3878b1fdae5a35 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Wed, 16 Oct 2024 13:55:17 +0200
Subject: [PATCH] arm64: dts: fix imx335 supplies properties in
 stm32mp257f-ev1.dts

Correct imx335 regulator nodes name which should be
such as regulator-[0-9]v[0-9].

Change-Id: I8e5f0c8ce9211c0a5df66479fd181d5f6f4cd4e8
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/411742
Reviewed-by: Alain VOLMAT <alain.volmat@st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
Tested-by: Alain VOLMAT <alain.volmat@st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index ad225a950dc1..f4032a1459fd 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -113,7 +113,7 @@ hdmi_connector_in: endpoint {
 		};
 	};
 
-	imx335_2v9: imx335-2v9 {
+	imx335_2v9: regulator-2v9 {
 		compatible = "regulator-fixed";
 		regulator-name = "imx335-avdd";
 		regulator-min-microvolt = <2900000>;
@@ -121,7 +121,7 @@ imx335_2v9: imx335-2v9 {
 		regulator-always-on;
 	};
 
-	imx335_1v8: imx335-1v8 {
+	imx335_1v8: regulator-1v8 {
 		compatible = "regulator-fixed";
 		regulator-name = "imx335-ovdd";
 		regulator-min-microvolt = <1800000>;
@@ -129,7 +129,7 @@ imx335_1v8: imx335-1v8 {
 		regulator-always-on;
 	};
 
-	imx335_1v2: imx335-1v2 {
+	imx335_1v2: regulator-1v2 {
 		compatible = "regulator-fixed";
 		regulator-name = "imx335-dvdd";
 		regulator-min-microvolt = <1200000>;
-- 
2.50.1

